README.md

```diff
@@ -299,6 +300,31 @@ import icon from '@assets/icon';
 import App from './App';
 ```
 
+##### 6. Enforce a blank line after top of file comments
+
+If you have pragma-comments at the top of file, or you have boilerplate copyright announcements, you may be interested in separating that content from your code imports. Explicitly providing `"<BUILTIN_MODULES>", "<THIRD_PARTY_MODULES>"` in your list will allow you to add a separator before them!
+
+```json
+"importOrder": [
+    "", // Include the separator here! And also provide "<BUILTIN_MODULES>", "<THIRD_PARTY_MODULES>" later
+    "<BUILTIN_MODULES>",     // This must be present in the list somewhere
+    "<THIRD_PARTY_MODULES>", // This must be present in the list somewhere
```

Comment:
What's the reason for the limitation that both builtins and third party need to be in the list for this to work?  Because if they're not there it's ambiguous what we should do, whether we should inject those magic strings before or after the newline?

Reply:
It’s a consequence of how we normalize the importOrder:
1. Check if `<THIRD_PARTY_MODULES>` is present, if not: inject it at front
2. Check if `<BUILTIN_MODULES>` is present, if not: inject at front
3. Check if first element is empty separator: if yes, set the flag `providesGapAfterTopOfFileComments`

Since we only know if you meant “after file-comments” if the first element is a separator, we need you to insert the special words so that you can put a separator before them.

I’m not sure how we could change normalization without having worse consequences?

Reply:
My only idea would be to check if the first element is a separator, and if so, inject third party and builtins under it.  But like I said, that might not be what the user wanted, because maybe they wanted a gap between those and their own code.  But, if they do, then they could add those special words explicitly, and honestly if that's their intent then specifying them explicitly is preferred for understandability.  

So, I have a minor preference to adjusting as I suggested, and making a blank line at the top always mean "space after top-of-file comments".  That just feels a bit easier to explain/understand.  What do you think?

Reply:
(Implemented, resolving discussion)